games-arcade/ceferino/ceferino-0.97.8-r2.ebuild

```diff
@@ -1,35 +1,34 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
+
 inherit autotools desktop
 
 DESCRIPTION="Super-Pang clone (destroy bouncing balloons with your grapnel)"
 HOMEPAGE="http://www.losersjuegos.com.ar/juegos/ceferino"
 SRC_URI="mirror://debian/pool/main/c/ceferino/${PN}_${PV}+svn37.orig.tar.gz"
+S+="+svn37"
 
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="nls"
 
 RDEPEND="
-	>=media-libs/libsdl-1.2[video]
-	>=media-libs/sdl-image-1.2
-	>=media-libs/sdl-mixer-1.2
+	media-libs/libsdl[sound,video]
+	media-libs/sdl-image[jpeg,png]
+	media-libs/sdl-mixer[mod]
 	nls? ( virtual/libintl )
 "
 DEPEND="${RDEPEND}
 	nls? ( sys-devel/gettext )
 "
 
-S="${WORKDIR}/${P}+svn37"
+PATCHES=( "${FILESDIR}"/${P}-fix-audio.patch )
```

Comment:
Unimportant but, even for a single file, it's typically preferred to:
```
PATCHES=(
    file
)
```
Makes it easier to add/change patches without restyling every time.